backend/guides/logging/how_to_add_log.md

```diff
@@ -0,0 +1,35 @@
+# How To Add Logs using the logger
+
+## Environmental Variables:
+
+- For local development, in backend/.env, set the environmental variable "MAX_LOG_LEVEL" to the desired value for the max level of log to show. i.e.
+  "MAX_LOG_LEVEL=debug" to show all 4 levels
+- For heroku deployment, got to Settings -> Config Vars, and similarly set the MAX_LOG_LEVEL.
+
+## Step-by-step Guide:
+
+To use the logger, first import it:
+
+```
+import logger from '@Logger';
+```
+
+Then to use it, do
+
+```javascript
+logger.log({
+  level: 'debug',
+  message: 'teapot info',
+  customTags: 'testing',
```

Comment:
customTags suggests that we can add multiple tags to a log message - would it be possible to look into that as well?

Reply:
Yes it's possible to add multiple tags. Details are in the docs.